src/pipeline/activator/cog-clip/README.md

```diff
@@ -0,0 +1,297 @@
+              "description":"A lower resolution thumbnail of the same image as the 'rgb' asset.",
+              "roles":[
+                 
+              ],
+              "type":"image/jpeg"
+           },
+           "flight_log":{
+              "href":"http://aviris.jpl.nasa.gov/cgi/flights_13.cgi?step=view_flightlog&flight_id=f130329t01",
+              "title":"flight_log",
+              "description":"HTML page with table listing the runs for this flight.",
+              "roles":[
+                 
+              ],
+              "type":"text/html"
+           },
+           "kml_outline":{
+              "href":"http://aviris.jpl.nasa.gov/aviris_locator/y13_KML/f130329t01p00r06_sc01_outline_KML.kml",
+              "title":"kml_outline",
+              "description":"KML file describing the flight outline",
+              "roles":[
+                 
+              ],
+              "type":"application/vnd.google-earth.kml+xml"
+           },
+           "ftp_refl":{
+              "href":"ftp://avoil:Gulf0il$pill@popo.jpl.nasa.gov/y13_data/f130329t01p00r06_refl.tar.gz",
+              "title":"ftp_refl",
+              "description":"AVIRIS data archive of atmospheric corrected imagery for this scene.",
+              "roles":[
+                 
+              ],
+              "type":"application/gzip"
+           }
+        },
+        "collection":"aviris-l2-cogs",
+        "properties":{
+           "YY":13,
+           "Run":6,
+           "Name":"f130329t01p00r06",
+           "Tape":"t01",
+           "Year":2013,
+           "Scene":"sc01",
+           "Flight":130329,
+           "GEO Ver":"ort",
+           "RDN Ver":"e",
+           "Comments":"Heading to southern end of So Cal Box Line 2",
+           "NASA Log":"132003",
+           "Rotation":73,
+           "datetime":"2013-03-29T18:17:00Z",
+           "has_refl":true,
+           "Flight ID":"f130329t01",
+           "Site Name":"Ad Hoc",
+           "Pixel Size":14.4,
+           "Flight Scene":"f130329t01p00r06_sc01",
+           "Investigator":"Robert Green",
+           "Solar Azimuth":140.15,
+           "Number of Lines":13139,
+           "Solar Elevation":52.9,
+           "File Size (Bytes)":9438628864,
+           "Number of Samples":1391,
+           "Max Scene Elevation":3019.12,
+           "Min Scene Elevation":122.25,
+           "Mean Scene Elevation":843.55,
+           "Gzip File Size (Bytes)":3659283368
+        }
+     }
+      ```
+</details>
+
+* `s3://aviris-data/test/f130329t01p00r06_corr_v1_warp.tif` is the test tiff.
+* FeatureCollection to test this module: [test.json.zip](https://github.com/azavea/nasa-hyperspectral/files/5798939/test.json.zip)
+* Set FeatureCollections as env variable (to simplify testing): `export FEATURES=`cat test-data/test.json`
+* Run clipping: `./sbt run clip --source-collection-id aviris-l2-cogs --source-item-id aviris_f130329t01p00r06_sc01 --source-asset-id cog --target-collection-id aviris-l2-chips`
```

Comment:
This command doesn't work. You need to quote the arguments like this:

```bash
./sbt run “clip --source-collection-id aviris-l2-cogs --source-item-id aviris_f130329t01p00r06_sc01 --source-asset-id cog --target-collection-id aviris-l2-chips”
```

After updating the documentation here, I'd consider running through everything with a fresh clone of the repository to make sure you're able to follow the instructions.